jena-text/src/main/java/org/apache/jena/query/text/TextQueryPF.java

```diff
@@ -295,6 +295,14 @@ private String chooseGraphURI(ExecutionContext execCxt) {
         return results;
     }
     
+    private boolean noLang(String lang) {
+        return (lang == null || lang.isEmpty() || " ".equals(lang));
```

Comment:
I don't know much about the text indexing, but this single space-- does that have some special meaning, or is it just a simple check for whitespace? I.e., could we see more than one space char here?

Reply:
The single space is the response from `o.getLiteralLanguage()` where `Node o` and `o.isLiteral()` at lines 344 and 386 of `TextQueryPF.java`. It appears to be the response when the literal has no explicit language tag. I thought the response was supposed to be an empty string, but the single space is what I was getting.

Reply:
Yes - no lang tag is a return of empty string.  If you get `<SPC>`, there is a bug somewhere.

Reply:
Other than this issue (" " for lang tag), the rest looks good.